packages/KIcon/icons/icn-brain.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you run these through https://jakearchibald.github.io/svgomg/ to shrink down/clean up?

Reply:
I already did. do we have any svg conventions that this one is breaking? it looks like most of the others have the same width and height.

Reply:
As long as they are only paths and run through SVGO you should be good to go. Didn’t mean to block just wanted to double check!

Also looks like the snapshot needs to be updated